libs/txnbuild/whitelist.go

```diff
@@ -36,6 +38,7 @@ func (whitelist *Whitelist) BuildXDR() (vxdr.VeloOp, error) {
 	return vxdr.VeloOp{Body: body}, nil
 }
 
+// FromXDR for Whitelist initialises the vtxnbuild struct from the corresponding xdr Operation.
```

Comment:
FromXDR for Whitelist initialises the vtxnbuild struct from the corresponding **XDR** Operation.